tests/test_qubes_rpc.py

```diff
@@ -0,0 +1,50 @@
+import io
+import os
+import unittest
+import yaml
+
+QUBES_POLICY_PREFIX = "/etc/qubes-rpc/policy/qubes."
+
+
+class SD_Qubes_Rpc_Tests(unittest.TestCase):
+
+    def setUp(self):
+        self.expected = self._loadVars()
+
+    def tearDown(self):
+        pass
+
+    def test_Policies(self):
+        # Using a for loop instead of pytest.parametrize due to
+        # the absence of pytest in dom0.
+        fail = False
+        for policy in self.expected:
+            if not self._startsWith(policy['policy'],
+                                    policy['starts_with']):
+                fail = True
+        self.assertFalse(fail)
+
+    def _startsWith(self, filename, expectedPolicy):
+        filePath = os.path.join(QUBES_POLICY_PREFIX + filename)
+        with io.open(filePath, 'r') as f:
+            actualPolicy = f.read()
+            if actualPolicy.startswith(expectedPolicy):
+                return True
+            else:
+                print("Policy for {} is:\n{}".format(filename,
+                                                     actualPolicy))
+                print("Policy for {} should be:\n{}".format(filename,
```

Comment:
Nit: more aggressive output would be helpful here for when the policies fail. Consider adding more newlines and header blocks like:

```
print("\n\n#### BEGIN RPC policy error report ####\n\n")
print("Policy for {} is:\n{}".format(filename,
                                                     actualPolicy))
print("Policy for {} should be:\n{}".format(filename,
                                                     expectedPolicy))
print("\n\n#### END RPC policy error report ####\n\n")
```